src/server/appserver.cpp

```diff
@@ -411,7 +402,7 @@ void AppServer::stopSyncTask(int syncDbId) {
     ASSERT(_syncPalMap[syncDbId].use_count() == 1)
     _syncPalMap.erase(syncDbId);
 
-    ASSERT(_vfsMap[syncDbId].use_count() == 1)
+    ASSERT(_vfsMap[syncDbId].use_count() <= 1)  // `use_count` can be zero when the local drive has been removed.
```

Comment:
This prevents a crash in debug mode, under the conditions describe the Jira ticket: the local drive is removed and the user remove the corresponding synchronization too.